### ISSUE TITLE
Cheat input

### ISSUE THREAD

**Issue by AidanDaniel97** [SkyTemple/py-desmume#11]
Just wondering if there is a way to interface with the chat input - adding/removing action replay codes specifically?

**Comment by theCapypara** [comment]
No, the only thing you could do is converting the AR code into a memory write hook.

**Comment by theCapypara** [comment]
Um sorry, memory read of course.
